pkg/agent/flowexporter/ipfix/ipfix_record.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipfix
+
+import (
+	"bytes"
+
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+)
+
+var _ IPFIXRecord = new(ipfixDataRecord)
+var _ IPFIXRecord = new(ipfixTemplateRecord)
+
+type IPFIXRecord interface {
```

Comment:
Comment here. In particular, explain why you are wrapping `ipfixentities.Record` from the go-ipfix library into an interface.

Reply:
done